libraries/lib-import-export/GetAcidizerTags.cpp

```diff
@@ -13,7 +13,7 @@
 #include <algorithm>
 #include <memory>
 
-namespace ModPcm
+namespace LibImportExport
```

Comment:
OK, I see you fixed the configuration problem.

This completes a round of review.